planex/cmd/makesrpm.py

```diff
@@ -240,11 +248,14 @@ def main(argv=None):
 
     args = parse_args_or_exit(argv)
     tmpdir = tempfile.mkdtemp(prefix="px-srpm-")
+    for subdir in [ 'BUILD', 'BUILDROOT','RPMS','SOURCES','SPECS','SRPMS' ]:
+        os.mkdir(os.path.join(tmpdir, subdir))
```

Comment:
Are these directories guaranteed not to exist? If not use `util.makedirs()`

Reply:
They are being made in the tmpdir created by tempfile.mkdtemp() in the line above, so unless tempfile.mkdtemp() has a rather serious problem, they are guaranteed not to exist.